streams/src/main/java/org/apache/kafka/streams/kstream/UnlimitedWindows.java

```diff
@@ -93,4 +93,8 @@ public int hashCode() {
         return (int) (start ^ (start >>> 32));
     }
 
+    @Override
+    public long maintainMs() {
```

Comment:
It will just effectively revert to compaction only. Which, i think, is what it should be for UnlimitedWindows.